docs/configuration.md

```diff
@@ -41,11 +41,6 @@ This option obviously would not make sense for language servers for other langua
 Here is a list of the additional settings currently supported by `haskell-language-server`, along with their setting key (you may not need to know this) and default:
 
 - Formatting provider (`haskell.formattingProvider`, default `ormolu`): what formatter to use; one of `floskell`, `ormolu`, `fourmolu`, `stylish-haskell`, or `brittany` (if compiled with the brittany plugin).
-- Format on imports (`haskell.formatOnImportOn`, default true): whether to format after adding an import.
-- Diagnostics on change (`haskell.diagnosticsOnChange`, default true): (currently unused).
-- Completion snippets (`haskell.completionSnippetsOn`, default true): whether to support completion snippets. *Deprecated* as it is equivalent to `haskell.plugin.ghcide-completions.config.snippetsOn`.
```

Comment:
the snippets setting still works

Reply:
`completionSnippetsOn` doesn't! The other one does... but this bit of doc doesn't list all the plugin settings. I'm dissatisfied with it for that reason (see the PR description), but for now I thought it was simplest to just get rid of this item.